android/app/src/main/res/drawable/ic_noti_bookmark.xml

```diff
@@ -0,0 +1,7 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I guess its not possible to use an xml for notifications directly.
for  this we need extra dependencies.
Will just update the other files.